src/DasMulli.Win32.ServiceUtils/ScAction.cs

```diff
@@ -8,7 +8,7 @@ namespace DasMulli.Win32.ServiceUtils
     /// <summary>
     /// Service control actions used to specify what to do in case of service failures.
     /// </summary>
-    /// <seealso cref="System.IEquatable{ScAction}" />
+    /// <seealso cref="System.IEquatable{T}" />
```

Comment:
There's (regrettably) no such thing in XML docs as referring to a constructed generic type, such as this one which appears to use the `ScAction` type as an argument. In reality what's happening is that the XML docs interpret this as a type definition (just `System.IEquatable<>`). Roslyn doesn't check the name of the type parameter, so you're aliasing it as though `System.IEquatable<>` was defined with a type parameter named `ScAction` instead of `T`.

I've clarified it here so that we're not abusing the semantics.

See `https://github.com/dotnet/csharplang/issues/401` and `https://github.com/dotnet/csharplang/pull/1751` for progress on this issue.